deploy/helms/legion/templates/jupyter/nfs-webserver.yaml

```diff
@@ -49,6 +49,18 @@ spec:
             mountPath: "/usr/share/nginx/html"
           - name: config-volume
             mountPath: "/etc/nginx/conf.d/"
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 80
+          initialDelaySeconds: 120
+          periodSeconds: 3
+        readinessProbe:
```

Comment:
Do we really need liveness and readiness probe in this case?

Reply:
addressed